conductor/lib/uploader_v2.py

```diff
@@ -467,10 +467,10 @@ def next_upload(self):
             uploads = Backend.next(
                 self.account, location=self.location,
                 project=self.project) or []
-        except BackendDown as err:
+        except exceptions.BackendDown as err:
```

Comment:
You can combine the two exceptions into
`except (exceptions.BackendDown, exceptions.BackendError,):`
and `raise` without passing `err`.